pkg/cli/environment_test.go

```diff
@@ -100,8 +100,8 @@ func TestNamespaceErr(t *testing.T) {
 	// doesn't matter, this was just the simplest way to force an error to
 	// occur. Users of this package are not able to do this, but the resulting
 	// behavior is the same as if any other error had occurred.
-	kConfigPath := "This doesn't even look like a valid path name"
-	env.config.KubeConfig = &kConfigPath
+	configPath := "This doesn't even look like a valid path name"
+	env.config.KubeConfig = &configPath
```

Comment:
Why is this being changed? Was this failing the linter?

Reply:
`kConfigMap` fails revive's var-naming rule with the following error: `var-naming: don't use leading k in Go names; var kConfigPath should be configPath (revive)`. [Here](https://github.com/golang/lint/issues/258) is a Github issue about this rule.

To avoid changing the variable name `kConfigMap`, 'k' could be added to a list of ignored characters for this rule.